modules/install/pages/sizing-general.adoc

```diff
@@ -261,6 +261,13 @@ The larger the fragmentation, the more disk you will need.
 
 The above index consumes 6 GB of disk space.
 
+=== Accommodating the Access Scanner
+
+The _access scanner_ is used to ensure high-performance data-loading, when buckets are restarted.
+The access scanner is highly CPU-intensive, and shares a thread pool with the Index Service: consequently, if scanning is performed simultaneously with index updates, the performance of the index updates may be adversely affected.
```

Comment:
Same as above, need to be clear what is actually being affected here - the index updates are performed elsewhere, but this can block the mutations from making out of the Data Service across the network to the Index Service.

Reply:
See previous comment - not sure I agree with "and shares a thread pool with the Index Service"